plugins/janus_videoroom.c

```diff
@@ -9173,6 +9188,7 @@ static void *janus_videoroom_handler(void *data) {
 						ps->acodec = JANUS_AUDIOCODEC_NONE;
 						ps->vcodec = JANUS_VIDEOCODEC_NONE;
 						ps->pt = -1;
+						ps->fmtp = NULL;
```

Comment:
This is not needed, `g_malloc0` allocates memory and sets it all to `0` to that variable is `NULL` already.